tools/make.py

```diff
@@ -697,11 +697,9 @@ def version_stamp_pboprefix(module,commitID):
         f.close()
 
         if configtext:
-            patchestext = re.search(r"version.*?=.*?$", configtext, re.DOTALL)
-            if patchestext:
+            if re.search(r"version=(.*?)$", configtext, re.DOTALL):
                 if configtext:
```

Comment:
This whole part is probably redundant, with centralised rvmake repository @glowbal started I think it's non-issue right now, can make an issue there.